### ISSUE TITLE
deployed on vercel often leads to timeout

### ISSUE THREAD

**Issue by allentofight** [mandarini/openai-assistant-demo#1]
by vercel's doc, "[Functions](https://vercel.com/docs/functions/edge-functions) on the [Edge Runtime](https://vercel.com/docs/functions/edge-functions/edge-runtime) do not have a max duration limit but must begin sending a response within 25 seconds. They require an [initial response](https://vercel.com/docs/concepts/functions/edge-functions/limitations#maximum-initial-response-time) then allow you to [continue streaming responses](https://vercel.com/docs/concepts/functions/edge-functions/streaming) after that time."

so you project works well on local, but cannot work on vercel, because it exceeds 25s very easily

**Comment by mandarini** [comment]
Yeah, @allentofight thanks for reporting this, I know :( I should've written it in the README. I actually upgraded my project so that I could increase the timeout of the edge function.
